packages/reason-expo/src/BlurView.re

```diff
@@ -1,10 +1,10 @@
 [@bs.module "expo-blur"] [@react.component]
 external make:
   (
-    ~tint: [@bs.string] [ | `default | `light | `dark]=?,
+    ~tint: [ | `default | `light | `dark]=?,
```

Comment:
Maybe here we want something like:

```
[@bs.deriving jsConverter]
type tint = [ | [@bs.as "default"] `default | [@bs.as "light"] `light | [@bs.as "dark"] `dark];

tintToJs(`default);
```

Not really sure how to fit jsConverter inside a binding.

Reply:
Since ReScript/BuckleScript 8.2 there is no need for converting between Reason and JS union representation as poly simple poly variants are just strings (like in Flow/TS) so this works out of the box.

Reference: https://rescript-lang.org/blog/string-literal-types-in-reason#string-literal-types-in-bindings

Reply:
Oh, I miss that we were using bs-platform 8.x!